test_docopt.py

```diff
@@ -65,13 +65,11 @@ def test_commands():
 
 
 def test_formal_usage():
-    doc = """
-    Usage: prog [-hv] ARG
-           prog N M
-
-    prog is a program."""
```

Comment:
In fact, this line was indented and therefore it was a part of the 'Usage' section. If the emty line is removed and lines are aligned (they all are already indented) then this section could be rewritten in the following way:
```
Usage: prog [-hv] ARG
       prog N M
       prog is a program.
```
which looks like a wrong description.